api/src/main/java/com/codeforcommunity/dto/site/FilterSiteImageResponse.java

```diff
@@ -0,0 +1,109 @@
+package com.codeforcommunity.dto.site;
+
+import java.sql.Date;
+
+public class FilterSiteImageResponse {
+
+  private int imageId;
+  private String imageUrl;
+  private int siteId;
+  private String uploaderName;
+  private String uploaderEmail;
+  private String dateSubmitted;
```

Comment:
⛏️ Can we convert the `dateSubmitted` field to a regular java `Date`, then add this decorator above it: `@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "MM/dd/yyyy")`
This will automatically convert the object into a string in the response body and let us easily change the format if we ever want to.